docs/airgap-install.md

```diff
@@ -57,34 +57,45 @@ Copy the `bundle_file` you created in the previous step or downloaded from the [
 
 As an alternative to the previous step, you can use k0sctl to upload the bundle file to the worker nodes. k0sctl can also be used to upload k0s binary file to all nodes. Take a look at this example (k0sctl.yaml) with one controller and one worker node to upload the bundle file and k0s binary:
 
-```YAML
+```yaml
 apiVersion: k0sctl.k0sproject.io/v1beta1
 kind: ClusterConfig
 metadata:
   name: k0s-cluster
 spec:
-  hosts:
-  - ssh:
-      address: <ip-address-controller>
-      user: ubuntu
-      keyPath: /path/.ssh/id_rsa
-    role: controller
-    uploadBinary: true
-    k0sBinaryPath: /path/to/k0s_binary/k0s
-  - ssh:
-      address: <ip-address-worker>
-      user: ubuntu
-      keyPath: /path/.ssh/id_rsa
-    role: worker
-    uploadBinary: true
-    k0sBinaryPath: /path/to/k0s_binary/k0s
-    files:
-      - name: bundle-file
-        src: /path/to/bundle-file/airgap-bundle-amd64.tar
-        dstDir: /var/lib/k0s/images/
-        perm: 0755
   k0s:
     version: 1.21.3+k0s.0
+  hosts:
+    - role: controller
+      ssh:
+        address: <controller-ip-address>
+        user: ubuntu
+        keyPath: /path/.ssh/id_rsa
+
+      #  uploadBinary: <boolean>
+      #    When true the k0s binaries are cached and uploaded
```

Comment:
Somehow I'm used to see the comments about settings above them, not below them.

Reply:
Yeah the first row was intended to be some kind of title for the doc block, not a template. This is more apparent in the comment for the k0sBinaryPath where the "template" has been commented out.